python/ml4ir/applications/ranking/tests/test_fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import warnings
+import tensorflow as tf
+from ml4ir.base.model.architectures.fixed_additive_positional_bias import FixedAdditivePositionalBias
+
+warnings.filterwarnings("ignore")
+
+
+class TestFixedAdditivePositionalBias(unittest.TestCase):
+
+    def calling_additive_positional_bias(self, rank_index, max_ranks, training):
+        """Testing to invoke additive positional bias"""
+        positional_bias = FixedAdditivePositionalBias(max_ranks)
+        biases = positional_bias(tf.convert_to_tensor(rank_index), training)
+        if not training:
+            assert tf.math.reduce_sum(biases).numpy() == 0.0
+        else:
+            for i in range(len(rank_index)):
+                assert biases[i][0].numpy() == positional_bias.weights[0][rank_index[i] - 1].numpy()[0]
+
+    def test_one_hot_conversion(self):
+        """calling additive positional bias"""
+        self.calling_additive_positional_bias([5, 4], 5, True)
```

Comment:
I am ramping-up on scala currently, and a pattern I see a lot is having an iterable and applying a function for each member of it. In scala it is elegant. In Python, we also have map etc. But here, I am wondering if it would have been more readable to write:
```python
for rank_index, max_ranks, does_training in zip([[1,3], [1,2], [2], [2,4,6,8]], [5,5,2,2,10], [True, False, True, False, True]): 
    self.calling_additive_positional_bias(rank_index, max_ranks, does_training)
```
It is a general comment this one, not a blocker.